song-server/src/main/java/bio/overture/song/server/service/ValidationService.java

```diff
@@ -62,23 +68,26 @@ public class ValidationService {
   private final UploadRepository uploadRepository;
   private final boolean enforceLatest;
   private final Schema analysisTypeIdSchema;
+  private final RestTemplate restTemplate;
```

Comment:
is there a way to use rather a `RetryTemplate`? just in case we require retry mechanism

Reply:
In order to apply Retry mechanism on the `RestTemplate` we should consider Timeout on UI as retry will take considerable amount of time. As a result, if the `RestTemplate` fails. means that the whole operations failed.